packages/docs/src/pages/guides/variants.mdx

```diff
@@ -162,7 +165,19 @@ These variants can then be used in the `theme.styles` object, with the `sx` prop
 />
 ```
 
-## Themeable Layout Components
+## `styles` variants
+
+Variants inside the `styles` object, while usable through the same mechanisms as regular variants,
+are also used for other Theme UI APIs.
+
+- The entire [`styles` object](/theme-spec#styles) can style child HTML elements or MDX content,
```

Comment:
Hey @lachlanjc, I'm not sure if this bullet doesn't "conflict" with the last one.
BaseStyles is used to style embedded HTML content, but MDX is styled with `Themed` (i.e. `##` maps to `Themed.h2`)

Reply:
Hmm. Using Next.js that's not the case (MDX is treated the same as HTML children, so you need BaseStyles), so I've never experienced that. I guess we can just remove the MDX reference for now?

Reply:
> MDX is treated the same as HTML children

That MDXProvider rendered by our ThemeProvider doesn't work in Next?

Reply:
Nope: https://github.com/system-ui/theme-ui/issues/1130

Reply:
Doesn't a new version of MDX packages specified in resolutions solve that issue?

https://github.com/system-ui/theme-ui/issues/1130#issuecomment-678727533

Reply:
Perhaps! Forgot about that & have never tried it

Reply:
> I guess we can just remove the MDX reference for now?

Yeah, this sound like a good idea :D

Reply:
https://github.com/system-ui/theme-ui/commit/fbb3451d43457ba041528c2bdb5ae9a602c8ff5b